### DISCUSSION TITLE
locked_balance is 0 after creating wallet using an existing cipher seed mnemonic

### DISCUSSION THREAD

**Discussion by youngzhenhao** [lightninglabs/taproot-assets#1206] [Q&A] [answered]
After I use an existing cipher seed mnemonic to create the wallet via `lncli create`, the "locked_balance" in `lncli walletbalance` is still 0, which means assets anchor sats is 0. But after I restore the `.tapd` files that I backed up, the assets are still available for use. Will this operation cause some potential problems?

**Comment by guggero** [comment, marked as answer]
The locked balance is for UTXOs that are reserved/leased/locked by `lnd` for internal operations such as sweeps. So it normally is zero. You would previously have seen a couple of thousand sats in imported accounts, that's how the asset UTXOs show up in `lnd`. Restoring the `lnd` wallet from the seed shouldn't be an issue (other than the "imported" balance not showing up automatically), as long as the seed is the same as you used previously for the `.tapd` folder in question.

**Reply by youngzhenhao** [reply]
Yes, I made a mistake. It's true that the imported balance instead of locked_balance didn't show up, so that is actually a normal situation.
